wrapper_create_env.py

```diff
@@ -0,0 +1,53 @@
+#! /bin/python
+import sys
+import subprocess
+import threading
+
+def call_shellscript(path, m_args, ssh_cmdline):
+	arglist = ssh_cmdline
+	arglist.append(path)
+	print arglist
+        print "the above is the cmdline that will run on the current machine"
+	for arg in m_args:
+		print arg
+		arglist.append(arg)
+	print "created argslist" + str(arglist)
```

Comment:
@suranapranay Ignore the files in this pull request. I will update them once I am done with coding. These all belong to old ones.